### ISSUE TITLE
Parser support

### ISSUE THREAD

**Issue by cbas** [ricardofbarros/linter-js-standard#15]
> standard supports custom JS parsers. To use a custom parser, install it from npm (example: npm install babel-eslint) and add this to your package.json:

> ```json
{
  "standard": {
    "parser": "babel-eslint"
  }
}
```

> https://github.com/feross/standard#can-i-use-a-custom-js-parser-for-bleeding-edge-es6-or-es7-support

I can't seem to get linter-js-standard to use this. Is there a way to do it?

Also seems to be related to #13 (reading `standard.ignore` setting from `package.json`). Maybe both can features be implemented the same way?

**Comment by ricardofbarros** [comment]
Nice :), I will ship this feature in the next major release, meaning it will come with #13. 

I hope to finish the #13 this next Saturday, so stay tune.

**Comment by cbas** [comment]
Thanks @ricardofbarros :) Big fan of this tool! :heart:

**Comment by ricardofbarros** [comment]
Glad you love it :smile:

**Comment by ricardofbarros** [comment]
@cbas The 1.0.0 is released. And supports this feature you requested, but only for `babel-eslint`.

So thats why I need your help. I need to add the custom parsers to dependencies, because Atom can't get the env variables, so it can't find packages installed globally. The only workaround acceptable is to install those parsers locally in the dependencies of the linter-js-standard. 

Do you know any other custom parsers? so I can add the support for them.

**Comment by cbas** [comment]
http://eslint.org/docs/user-guide/configuring.html#specifying-parser
> The following parsers are compatible with ESLint:
- [Esprima](https://npmjs.com/package/esprima)
- [Esprima-FB](https://npmjs.com/package/esprima-fb) - Facebook's fork of Esprima that includes their proprietary syntax additions.
- [Babel-ESLint](https://npmjs.com/package/babel-eslint) - A wrapper around the [Babel](http://babeljs.io/) parser that makes it compatible with ESLint.

**Comment by ricardofbarros** [comment]
Cool thank you, will include those 2 as well.

**Comment by ricardofbarros** [comment]
Okay just released v1.0.5, now it supports as well esprima and esprima-fb.

Thank you :+1:
